openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -279,12 +291,15 @@ def is_flag_active(self, flag_name, check_before_waffle_callback=None, flag_unde
                         value = flag_is_active(request, namespaced_flag_name)
                     else:
                         log.warning(u"%sFlag '%s' accessed without a request", self.log_prefix, namespaced_flag_name)
+                        set_custom_metric('warn_flag_no_request', True)
```

Comment:
**Reviewer Note:** This is a new metric to show that we've entered this special case, in case we get unexpected results in the existing flag metric.